docs/software/koji-workflow.md

```diff
@@ -47,53 +47,137 @@ The results of a buildArch task. Their metadata are recorded in the Koji databas
+
+Assign the ticket to the Software team.
+
+
+Initial Setup
+-------------
+
+You will be using the [OSG Build Tools](/software/osg-build-tools) to interact with Koji.
+You can use them on either your own machine or on your UW Comp Sci login node such as `moria`.
+
 
-Using Koji requires:
+### Setting up on moria
 
--   `osg-build` version 1.6.3 or later.
--   `koji` 1.6.0-2.osg or later. **Note** that you want a koji build from osg; the output of `rpm -q koji` should end in ".osg".
+Perform the following to set up the build tools on `moria`:
 
-Both pieces of software are available from the osg repositories. `osg-build` may also be obtained from GitHub by cloning out `https://github.com/opensciencegrid/osg-build`
+1.  Clone the osg-build git repo
 
-#### Special instructions for UW-Madison CSL machines:
+        :::console
+        you@moria$ git clone https://github.com/opensciencegrid/osg-build $HOME/osg-build
 
--   Clone the osg-build GitHub repo:
+1.  Set your `$PATH`:
 
         :::console
-        [you@host]$ git clone https://github.com/opensciencegrid/osg-build
+        you@moria$ export PATH=$PATH:$HOME/osg-build
+        you@moria$ export PATH=$PATH:/p/vdt/workspace/quilt/bin
+        you@moria$ export PATH=$PATH:/p/vdt/workspace/tarball-client/stable/sys
+
+1.  Copy your user certificate and key into `$HOME/.globus/usercert.pem` and `$HOME/.globus/userkey.pem`.
+    Make sure `userkey.pem` is only readable by yourself.
+
+1.  (Optional) Load your certificate into your browser.
+    This will allow you to make some changes using the [Koji web interface](https://koji.opensciencegrid.org/koji).
 
--   Add this directory to your `$PATH`
--   Run
+1.  Set up the OSG Koji config
 
         :::console
-        [you@host]$ osg-koji setup
+        you@moria$ osg-koji setup
 
-    to set up the koji configuration and certificates in `~/.osg-koji`
+    Answer "yes" to all questions.
 
-### Obtaining a login
 
-You will be using your grid certificate to log in. Email a Koji admin the DN of your certificate, and we will set up a Koji account with the appropriate permissions.
+### Setting up on your own host
 
-If you are switching certificate providers, you will need to email a Koji admin with your new DN. You will also need to clear your browser cookies and cache for `https://koji.opensciencegrid.org` before trying to use the Koji web interface again. If your CN has changed, you will not be able to use your old certificate.
+This requires an Enterprise Linux 6 or 7 host.
 
-Current Koji admins are Mat Selmeci and Carl Edquist.
+1.  Install the [OSG YUM repositories](https://opensciencegrid.org/docs/common/yum/)
 
-### Configuring certificate authentication
+1.  If using OSG 3.5 or newer, enable the `devops` repository.
 
-You must also configure certificate authentication for the command-line tools on your build host:
+1.  Install osg-build and its dependencies:
 
--   Run
+        :::console
+        you@host$ sudo yum install osg-build
```

Comment:
Is it noted somewhere, or worth noting, that running from a git repo is still an option for "your machine" also?